test/functional/__snapshots__/markdown-full.spec.js.snap

```diff
@@ -0,0 +1,64 @@
+exports[`createMarkdown outputs correct changelog 1`] = `
+"
+## Unreleased (2099-01-01)
+
+#### :rocket: New Feature
+* \`the-force-awakens\`, \`rogue-one\`
```

Comment:
It's just what the snapshot output looks like, the normal output doesn't have the escapes.